ee/billing/billing_manager.py

```diff
@@ -179,7 +179,7 @@ def get_billing(self, organization: Optional[Organization], plan_keys: Optional[
                     product["tiers"] = compute_usage_per_tier(
                         limited_usage, product["projected_usage"], product["tiers"]
                     )
-                    product["current_amount_usd"] = sum_total_across_tiers(product["tiers"])
+                    product["current_amount_usd"] = str(sum_total_across_tiers(product["tiers"]))
```

Comment:
This has to be a string because of the parseFloat conversion we do on the frontend. parseFloat will return NaN if it receives an integer input, so it's safest to string cast. 

The underlying function uses Decimal, but it seems that Decimal(0) gets converted to int after getting json encoded.